server/src/main/java/io/spine/server/storage/memory/EntityQueryMatcher.java

```diff
@@ -123,14 +123,14 @@ private static boolean checkAll(Multimap<EntityColumn, ColumnFilter> filters,
     }
 
     private static boolean checkEither(Multimap<EntityColumn, ColumnFilter> filters,
-                                       Map<String, MemoizedValue> entityColumns) {
+                                       EntityRecordWithColumns record) {
         for (Map.Entry<EntityColumn, ColumnFilter> filter : filters.entries()) {
-            final String columnName = filter.getKey()
-                                            .getName();
-            final MemoizedValue memoizedValue = entityColumns.get(columnName);
-            final boolean matches = checkSingleParameter(filter.getValue(), memoizedValue);
-            if (matches) {
-                return true;
+            final Optional<MemoizedValue> columnValue = getColumnValue(record, filter.getKey());
+            if (columnValue.isPresent()) {
```

Comment:
Please cover the `false` branch as well.

Reply:
Done.